vunit/test_runner.py

```diff
@@ -210,6 +210,8 @@ def read_output():
             results = test_suite.run(output_path=output_path,
                                      read_output=read_output)
         except KeyboardInterrupt:
+            results[test_suite.name] = SKIPPED
```

Comment:
It is not test_suites but rather test_cases within test_suites that have test report results. When running several tests within the same simulation there are multiple test_cases within a test_suite. See the code that prints the test status using a for loop over test cases within the test suite.